plugins/oaiMetadataFormats/marc/OAIMetadataFormat_MARC.php

```diff
@@ -51,18 +53,19 @@ public function toXml($record, $format = null)
             'article' => $article,
             'publication' => $publication,
             'issue' => $record->getData('issue'),
-            'section' => $record->getData('section')
+            'section' => $record->getData('section'),
+            'publicationLocale' => $publicationLocale,
         ]);
 
         $subjects = array_merge_recursive(
             stripAssocArray((array) $publication->getData('discipline')),
-            stripAssocArray((array) $publication->getData('subject'))
+            stripAssocArray((array) $publication->getData('subjects'))
```

Comment:
The untyped properties attack us again, we should have left the getters/setters, hopefully we'll get rid of the `setData()`/`getData()` soon 🥲